healthcareai/tests/test_dev_supervised_model_class.py

```diff
@@ -82,8 +109,14 @@ def test_linear_dev_tune_false(self):
         df.drop(['PatientID', 'InTestWindowFLG'], axis=1, inplace=True)
 
         np.random.seed(42)
-        o = DevelopSupervisedModel(dataframe=df, model_type='classification', predicted_column='ThirtyDayReadmitFLG')
-        o.data_preparation(impute=True)
+        df = pipelines.dataframe_preparation_pipeline(df,
+                                                      CLASSIFICATION,
+                                                      GRAIN_COLUMN_NAME,
+                                                      PREDICTED_COLUMN,
+                                                      impute=True)
+        o = DevelopSupervisedModel(df, CLASSIFICATION, PREDICTED_COLUMN)
+
+        o.train_test_split()
         o.linear(cores=1)
 
         self.assertAlmostEqual(np.round(o.au_roc, 3), 0.672000)
```

Comment:
I assume we'll dig through the test functionality a little more completely when you are done with all the functionality...
I didn't check them carefully.